templates/components/common/responsive-img.html

```diff
@@ -0,0 +1,50 @@
+{{!--
+This file implements the lazysizes srcset pattern (as seen on
```

Comment:
I'm wondering if it's worth adding it to `paper-handlebars` so it can be used in other themes as well? 🤔

Reply:
Actually, I just saw your PR https://github.com/bigcommerce/paper-handlebars/pull/42 so I guess that could work as well 😄